Applications/ReferenceServer/Quickstarts.ReferenceServer.Config.xml

```diff
@@ -76,6 +76,7 @@
     <BaseAddresses>
       <ua:String>https://localhost:62540/Quickstarts/ReferenceServer</ua:String>
       <ua:String>opc.tcp://localhost:62541/Quickstarts/ReferenceServer</ua:String>
+      <ua:String>opc.tcp://localhost:62542/Quickstarts/ReferenceServer</ua:String>
```

Comment:
For 'production' config the extra endpoint should not be added?

Reply:
Yes we could leave it as it was :)